project/project.ts

```diff
@@ -369,7 +371,7 @@ export const makeTsGdProject = async (
   return new TsGdProjectClass(watcher, initialFiles, program, ts2gdJson, args)
 }
 
-const shouldIncludePath = (path: string, stats: fs.Stats): boolean => {
+const shouldIncludePath = (path: string, stats?: fs.Stats): boolean => {
```

Comment:
Just wondering when this is invoked?
Is this invoked for a deleted file on unlink? 
Or when would we have non-existing files here?

Reply:
chokidar will call this (indirectly) to see if it should care about certain files or not. It probably gets called when you delete/remove a file because it's checking to see if it should emit events about them.